lib/src/components/NxPageHeader/NxPageHeader.tsx

```diff
@@ -17,6 +17,7 @@ const logoImg = importImage('sonatype-logo-with-hexagon.png');
 function HeaderProductInfo({ name, version }: ProductInfo) {
   return (
     <>
+      <div role="presentation" className="nx-product__divider">|</div>
```

Comment:
This is no longer being done as a `::before`` because it made the styling undesirably complicated, as this bar needed a different top margin than the name.